plugins/nv-filters/nvafx-load.h

```diff
@@ -0,0 +1,340 @@
+#pragma once
+#include <Windows.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <util/platform.h>
+#include <util/windows/win-version.h>
+
+#define NVAFX_API
+#define MIN_VFX_SDK_VERSION (0 << 24 | 7 << 16 | 2 << 8 | 0 << 0)
+#define MIN_AFX_SDK_VERSION (1 << 24 | 3 << 16 | 0 << 0)
```

Comment:
Instead of having to update this in two places, does it make sense to have a small header of just these values that is included in both this file and `nvvfx-load.h`?